.github/workflows/main.yml

```
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
+
+name: tweet_scheduler
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  #schedule:
+  #  - cron: '30 10,18 * * *'    
+
+jobs:
+  hadleyfreebot-post:
+    runs-on: macOS-latest
```

why mac os latest VM? why not linux or windows?

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

```suggestion
    runs-on: ubuntu-latest
```